jobs/config.json

```diff
@@ -13086,6 +13086,27 @@
       "sig-scheduling"
     ]
   },
+  "pull-release-null-e2e": {
+    "args": [
+      "--build=bazel",
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/pull-kubernetes-e2e.env",
+      "--extract=local",
+      "--gcp-node-image=gci",
+      "--gcp-project=k8s-jkns-pr-gce-etcd3",
+      "--gcp-zone=us-central1-f",
+      "--provider=gce",
+      "--runtime-config=batch/v2alpha1=true,admissionregistration.k8s.io/v1alpha1=true",
+      "--stage=gs://kubernetes-release-pull/ci/pull-release-null-e2e",
+      "--test_args=--ginkgo.focus=",
```

Comment:
do `--ginkgo.focus=\\[Feature:Empty\\]` I think, or else it will use some default

Reply:
done